lib/apiservers/engine/backends/container.go

```diff
@@ -345,12 +334,115 @@ func (c *ContainerBackend) ContainerExecResize(eid string, height, width int) er
 	return err
 }
 
+// attachHelper performs some basic type transformation and makes blocking call to AttachStreams
+// autoclose determines if stdin will be closed when both stdout and stderr have closed
+func (c *ContainerBackend) attachHelper(op trace.Operation, ec *models.TaskInspectResponse, stdin io.ReadCloser, stdout, stderr io.Writer, autoclose bool) error {
+	defer trace.End(trace.Begin(ec.ID))
+
+	ca := &backend.ContainerAttachConfig{
+		UseStdin:  ec.OpenStdin,
+		UseStdout: ec.OpenStdout,
+		// set UseStderr to false for Tty case as we merge stdout and stderr
```

Comment:
I think either `UseStderr` should explicitly be set to `false` here, or maybe reword this comment a little more clearly to cover why it is assumed that `ec.Tty` is always true.